tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -93,3 +104,27 @@ Verify List Of Libraries In Image
     Append To List    ${status_list}    ${status}
     Run Keyword If    '${status}' == 'FAIL'    Fail    Shown and installed libraries for ${image} image do not match
 
+Verify Errors In Logs
+    [Documentation]    Verifies that there are no errors in Logs
+    @{pods} =    Oc Get    kind=Pod    namespace=redhat-ods-applications
```

Comment:
If you are looking for the disutil error you have to search only in the pods with name starting with jupyterhub-1-*, not in all redhat-ods-applications pods

Reply:
Updated

Reply:
Updated